float32 top_Med
float32 top_mid
float32 top_midlat
float32 top_lat
float32 midtop_Med
float32 midtop_mid
float32 midtop_midlat
float32 midtop_lat
float32 mid_Med
float32 mid_mid
float32 mid_latUP
float32 mid_latDO
float32 midbot_Med
float32 midbot_lat
float32 bot_Med
float32 bot_lat
float32 all_force
bool stance_flg